modules/book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state
+
+  def initialize(date, publisher, cover_state, archived: false, id: nil)
+    super(id, date, archived: archived)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
```

Comment:
- [ ] In the initialize method I think it would be better if the id should be generated randomly e.g using `rand()` method. This would make our application function better.